tasks/mock-github

```diff
@@ -19,6 +19,7 @@ from task.test_mock_server import MockHandler, MockServer
 
 repo = None
 sha = None
+log = None
```

Comment:
I wonder if you'd like to add some more types...

Reply:
TBH not today when we have a crapton of stuff to catch up with. I'd like to reserve this to the next weeks -- I also want to rewrite run-local.sh as proper pytest.